### PR TITLE
refactor: unable to load followups on Artwork page

### PR DESCRIPTION
**Pull request by gkartalis**
This PR resolves [] 

### Description

Some followups for the Artwork page:

- Add `@principalField` to Artwork page after the audit
- defensive code on inquiryModal that was previously breaking the Artwork page

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [x] I hid my changes behind a **[feature flag]**, or they don't need one.
- [x] I have included **screenshots** or **videos**, or I have not changed the UI.
- [x] I have added **tests**, or my changes don't require any.
- [x] I added an **[app state migration]**, or my changes do not require one.
- [x] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [x] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-  unable to load followups on Artwork page - gkartalis

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Dev changes (unable to load followups on Artwork page - gkartalis)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 02c6ccfbd2397cc8706648ed853837ce49743f53
</p>

File: src/app/Scenes/Artwork/Components/CommercialButtons/InquiryModal.tsx [hunk 1]
```diff
@@ -276,7 +276,7 @@ export const InquiryModal: React.FC<InquiryModalProps> = ({ artwork, ...props })
           <Text variant="sm">What information are you looking for?</Text>
           {
             // NOTE: For now the inquiryQuestions field values are always present and therefore never null, so it is safe to destructure them
-            questions!.map((inquiryQuestion) => {
+            questions?.map((inquiryQuestion) => {
```
Comment (gkartalis) [inline src/app/Scenes/Artwork/Components/CommercialButtons/InquiryModal.tsx:279]: This should never be null apparently but in staging is broken and it breaks this Artwork page.

File: src/app/Scenes/Artwork/Artwork.tsx [hunk 2]
```diff
@@ -629,7 +629,7 @@ export const ArtworkContainer = createRefetchContainer(
 
 export const ArtworkScreenQuery = graphql`
   query ArtworkAboveTheFoldQuery($artworkID: String!) {
-    artwork(id: $artworkID) {
+    artwork(id: $artworkID) @principalField {
```
Comment (gkartalis) [inline src/app/Scenes/Artwork/Artwork.tsx:632]: Adds `@principalField` to the top level of the artwork query

File: src/app/Scenes/Artwork/Components/CommercialButtons/InquiryModal.tsx (hunk 1 above)
Comment (gkartalis) [reply src/app/Scenes/Artwork/Components/CommercialButtons/InquiryModal.tsx:279]: simulator_screenshot_454B9E58-D59B-462C-A70F-F522203E4A58

**Review by MounirDhahri — APPROVED** [review]
thanks for the quick fix!

File: src/app/Scenes/Artwork/Components/CommercialButtons/InquiryModal.tsx (hunk 1 above)
Comment (brainbicycle) [reply src/app/Scenes/Artwork/Components/CommercialButtons/InquiryModal.tsx:279]: should we get rid of the comment?

Comment (brainbicycle) [reply src/app/Scenes/Artwork/Components/CommercialButtons/InquiryModal.tsx:279]: I read it as justifying the `!` currently which is now gone 🤷 but also appears to be false if we are getting a red screen here, personally prefer not commenting to misleading comments

**Pull request merged**